core/blockchain.go

```diff
@@ -1912,6 +1945,22 @@ func (bc *BlockChain) WriteShardStateBytes(db rawdb.DatabaseWriter,
 	return decodeShardState, nil
 }
 
+func (bc *BlockChain) StoreShardStateBytes(epoch *big.Int, shardState []byte) (*shard.State, error) {
```

Comment:
can you reuse the existing WriteShardStateBytes() instead ?

Reply:
Technically yes, but it would be inconvenient.